pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/patch/Patch.java

```diff
@@ -123,6 +137,11 @@ public static boolean patch(@NonNull String id, @NonNull String user, @NonNull N
         }
         SimpleResult results = inTransaction((conn) -> {
             SimpleResult dbVals = new SimpleResult();
+            if (dbElement.isUniqueField) {
+                if(!isValueUnique(dbElement, String.valueOf(nameValue.getValue()))){
```

Comment:
For this to work as expected, we need to manually drop the unique constraints on the table

Reply:
You mean via Liquibase, rght?